packages/gatsby-plugin/README.md

```diff
@@ -0,0 +1,42 @@
+# gatsby-plugin-builder
+
+Plugin for integrating [Builder.io](https://www.builder.io) to allow drag and drop page building with Gatsby. It puts the Builder.io schema under an `allBuilderModels` field of the Gatsby GraphQL query, If a templates map is passed as option, this plugin will create gatsby pages dynamically for each page entry in builder.io on the path specified.
+
+## Install
+
+`npm install @builder.io/gatsby-plugin`
+
+## How to use
+
+```javascript
+// In your gatsby-config.js
+const path = require("path")
+module.exports = {
+  plugins: [
+    {
+      resolve: "@builder.io/gatsby-plugin",
+      options: {
+        // public API Key
+        publicAPIKey: "MY_PUBLIC_API_KEY",
+        templates: {
+          myPageModel: path.resolve("templates/my-page.tsx"),
```

Comment:
let's call this `page:` since that's the default, and have a comment that this is the name of your page model